jobs/webgroup/blockbot.hcl

```diff
@@ -21,7 +21,7 @@ job "blockbot" {
       template {
         data = <<EOF
 TOKEN={{ key "blockbot/discord/token" }}
-DEBUG=false
+DEBUG= # empty means false
```

Comment:
Why not just declare it as False. If you are adding an extra comment to say that it's false then just declare it as false.